app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def show; end
+
+  def edit; end
+
+  def update; end
+end
```

Comment:
- [ ] Same here as well, since you're using devise gem, you don't need a users_controller unless you intend to customize its functionalities. 👍